lib/Util.py

```diff
@@ -1,65 +1,47 @@
 # Contains all generic functions separate from the main logic
 # These are shared between all libraries and the main script
-from datetime import datetime #< Used to print the current time
-import sys #< Used to flush STDOUT or exit
-import os #< Check if a filepath is valid
-import web3 #< Handling wallet addresses
-# Import our own libraries
-from lib import Contract, State
 
-"""
-@brief Logs `info` to the terminal with an attached datetime
-@param log_level: gets filtered if above the configured `verbosity` value
-"""
-def log(info, log_level = 1):
-    level_string = ""
-    match log_level:
-        case 1:
-            level_string = "WARN"
-        case 2:
-            level_string = "INFO"
-        case 3:
-            level_string = "DEBUG"
-        case _:
-            level_string = "😕"
-    if (log_level > State.LOG_VERBOSITY):
-        return
-    if State.LOG_TIMESTAMPED:
-        now = datetime.now()
-        now_trimmed = now.strftime('%Y-%m-%d %H:%M:%S.%f')[:-3]
-        print("[{0}] {1} - {2}".format(now_trimmed, level_string, info))
-    else:
-        print("{0} - {1}".format(level_string, info))
-    sys.stdout.flush()
```

Comment:
I don't know what's the purpose of writing own `log()` function. python's `logging` is really full-featured. I don't believe we need `sys.stdout.flush()`... and loglevels are built in `logging`.